FATN_to_USB/fatn_to_usb/Get_FATN_Dropbox_link.py

```
@@ -144,17 +145,22 @@ def Download_FATN(download_url: str,
+if __name__ == '__main__':
+    """
+    Example usage when run directly.
+    This demonstrates the complete download workflow.
+    """
+    url = 'https://www.fatntalkingnews.org.uk/about-fatn-talking-news/#coronavirus'
+    FATN_parse = GetFATNUrl()
 
-FATN = urlopen(url)  # get the FATN wordpress webpage
-FATN_parse.feed(str(FATN.read()))  # parse it for the dropbox link
-# print(FATN_parse.FATNurl)
+    FATN = urlopen(url)  # get the FATN wordpress webpage
```

Ruff (0.14.8)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)